src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/corporates/CorporateResource.kt

```diff
@@ -30,6 +33,65 @@ import java.time.LocalDate
 @Validated
 @RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
 class CorporateResource(private val corporateService: CorporateService) {
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_CONTACTPERSONS')")
```

Comment:
moved this to the top so I can order stuff as:
- migrations stuff
- create the thing
- get the thing
- update the thing
- delete the thing